packages/salesforcedx-vscode-apex/src/commands/forceApexExecute.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2017, salesforce.com, inc.
+ * Copyright (c) 2021, salesforce.com, inc.
```

Comment:
As we check in files, should we be updating the copyright year?

Reply:
We usually only update this for new files. I don't know even if that matters.